src-vue/src/plugins/store.ts

```diff
@@ -43,6 +45,7 @@ export const store = createStore<FlightCoreStore>({
     state(): FlightCoreStore {
         return {
             developer_mode: false,
+            repair_view_visible: false,
```

Comment:
Should building in _debug mode_ (not to be confused with _dev mode_ which is activated by clicking version number 5 times), also show RepairView in menu bar by default like it does for DevView?